config/initializers/fb_jwt_auth.rb

```diff
@@ -1,5 +1,5 @@
 Fb::Jwt::Auth.configure do |config|
-  config.issuer = 'fb-runner'
+  config.issuer = ENV['SERVICE_SLUG']
```

Comment:
didn't we talk about potentially retrieving this from the service metadata? we don't currently set it there but it seems like a reasonable thing to do

Reply:
Actually this line can be removed in favor of the service_access_token method on the submitter adapter class. I will remove it